### PR TITLE
illustrate how to run go tests locally

### LINKED ISSUE #405: how to locally run the e2e test
## Expected Behavior

demonstrates for how to run e2e test locally with **required env**

## Actual Behavior

empty of https://github.com/knative/build/blob/master/test/README.md#environment-requirements

## Additional Info

ref serving part: https://github.com/knative/serving/blob/master/test/README.md#environment-requirements

### PR DESCRIPTION
**Pull request by zrss**
Fixes #405 

## Proposed Changes

  * illustrate how to run go tests locally

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ _By default `go test` will not run [the e2e tests](#running-end-to-end-tests), w
 
 ## Running end to end tests
 
+### Environment requirements
```
Comment (vdemeester) [inline test/README.md]: This is *duplicate* with https://github.com/knative/build/pull/407/files#diff-c30ff793110b0960f76bb3854af6f5a8R56, we should merge those.

File: test/README.md [hunk 2]
```diff
@@ -17,10 +17,18 @@ _By default `go test` will not run [the e2e tests](#running-end-to-end-tests), w
...
+
+1. Configure `ko` to point to your registry. [see here](https://github.com/knative/build/blob/master/DEVELOPMENT.md#one-time-setup)
+
 ### Go e2e tests
 
 To run [the Go e2e tests](./e2e), you need to have a running environment that meets
-[the e2e test environment requirements](#environment-requirements), and you need to specify the build tag `e2e`.
+[the e2e test environment requirements](#environment-requirements) and [stand up a version of this controller on-cluster](https://github.com/knative/build/blob/master/DEVELOPMENT.md#standing-it-up)
+
+finally run the Go e2e tests with the build tag `e2e`.
```
Comment (vdemeester) [inline test/README.md]: Finally (and a `.` above)

File: test/README.md (hunk 1 above)
Comment (zrss) [reply test/README.md]: oh, i am negligent and do some changes. PTAL

**Pull request merged**